Header header

# report the last head pan command sent to the dynamixels
float32 pan_command

# report the last head tilt command
float32 tilt_command
